### PR TITLE
Update license to attribute Plaid Technologies

### PR DESCRIPTION
**Pull request by pbernasconi**
@chrisforrette –the Plaid team would like to thank you for your contributions to the plaid-python library.

We are simply correcting the copyright attribution to reflect that ownership. Thanks again, so much, for your contributions! We have attributed your name in the README, as noted below.

### CODE REVIEW

**Comment by chrisforrette** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -349,7 +349,8 @@ $ make test
 ```
 
 ## Contributors
-- [PK](https://github.com/gae123) - fixes and Google App Engine Support
+- @chrisforrette (Chris Forrette)
+- @gae123
```
Comment (davidchambers) [inline README.md]: Let's make these links. :)

Comment (pbernasconi) [inline README.md]: ⚡

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+The MIT License
 
-Copyright (c) 2014 Chris Forrette
+Copyright (c) 2013-present Plaid Technologies, Inc. <support@plaid.com>
```
Comment (davidchambers) [inline LICENSE]: `s/2013-present/2016/`

Comment (pbernasconi) [inline LICENSE]: ⚡

**Comment by davidchambers** [comment]
:deciduous_tree:

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+The MIT License
 
-Copyright (c) 2014 Chris Forrette
+Copyright (c) 2013-2016 Plaid Technologies, Inc. <support@plaid.com>
```
Comment (davidchambers) [inline LICENSE]: 2013? The [initial commit][1] was in 2014.


[1]: https://github.com/plaid/plaid-python/commit/eaefc152be977635ed7327b8d3a8494ae80ef8b2

Comment (pbernasconi) [inline LICENSE]: ⚡

**Comment by davidchambers** [comment]
:deciduous_tree:

**Pull request merged**